release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -56,6 +56,70 @@ link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShif
 
 This release adds improvements related to the following components and concepts.
 
+[id="ocp-4-6-web-console"]
+=== Web console
+
+[id="ocp-4-6-web-console-improved-upgrade-experience"]
+==== Improved upgrade experience in the web console
+
+* Administrators are now better informed about the differences between the upgrade
+channels via helpful text and links in the web console.
+* Changes such as bug fixes and enhancements are now included in each minor and
+patch release.
+* There is now a visual representation of the different upgrade paths.
+* Special Alerts now inform administrators when new patch releases, new minor
+releases, and news channels become available.
+
+[id="ocp-4-6-web-console-improved-operator-installation-workflow"]
+==== Improved Operator installation workflow with OperatorHub
+
+When administrators install Operators with OperatorHub, they now get immediate
+feedback to ensure that the Operator is installing properly.
+
+[id="ocp-4-6-web-console-improved-operand-details-view"]
+==== Improved Operand details view
+
+You can now see the schema grouping of `specDescriptor` fields and the status of
+your Operands on the Operand's details view, so that you can easily see the
+status and configure the `spec` of the Operand instance.
+
+[id="ocp-4-6-web-console-view-operators-related-objects"]
+==== View cluster Operators' related objects
```

Comment:
Though technically fine, I'd suggest rewording to "View related objects for cluster Operators"